client/core/core.go

```diff
@@ -1460,6 +1460,30 @@ func (c *Core) ReconfigureWallet(appPW []byte, assetID uint32, cfg map[string]st
 	details := fmt.Sprintf("Configuration for %s wallet has been updated.", unbip(assetID))
 	c.notify(newWalletConfigNote("Wallet Configuration Updated", details, db.Success, wallet.state()))
 
+	// Clear any existing tickGovernors for suspect matches.
+	c.connMtx.RLock()
+	defer c.connMtx.RUnlock()
+	for _, dc := range c.conns {
+		dc.tradeMtx.RLock()
+		for _, t := range dc.trades {
+			if t.Base() != assetID && t.Quote() != assetID {
+				continue
+			}
+			t.mtx.Lock()
+			for _, m := range t.matches {
+				if m.tickGovernor != nil &&
+					(m.suspectSwap && t.wallets.fromAsset.ID == assetID ||
+						(m.suspectRedeem && t.wallets.toAsset.ID == assetID)) {
```

Comment:
Missing parenthesis around the `m.suspectSwap && t.wallets.fromAsset.ID == assetID`.

Perhaps this would read better like:

```go
	isFromAsset := t.wallets.fromAsset.ID == assetID
	for _, m := range t.matches {
		if m.tickGovernor != nil &&
			((m.suspectSwap && isFromAsset) || (m.suspectRedeem && !isFromAsset)) {
```